content/release-10-10-0/oee-10-10-0-bundle/10_10_0_3.md

```diff
@@ -0,0 +1,182 @@
+<td style="text-align:left">COE-541</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The OEE application was updated to Angular 11 and Cumulocity WebSDK 1011.0.15</td>
+<td style="text-align:left">COE-477</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Upgraded Apama to 10.7.2.5.</td>
+<td style="text-align:left">COE-556</td>
+</tr>
+</tbody>
+</table>
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">When using calculation categories, for all categories the formula of the first category was used, leading to incorrect calculation results. The calculation of categories has now been fixed.</td>
+<td style="text-align:left">COE-584</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">When using line profiles, the subscriptions for MEAs were not created correctly and OEE calculation was not performed. This is issue is now resolved, subscriptions are created for the calculation profiles to fix the OEE calculation of line profiles.</td>
+<td style="text-align:left">COE-532</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Formulas defining machine status and quality status conditions were not properly deleted. This led to problems when switching machine status and quality status calculation off. A similar problem occurred when enabling categories on a profile using machine and quality status calculations. This issue has been fixed, the formulas are now deleted correctly.</td>
+<td style="text-align:left">COE-543</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Using <i>Measurement count</i> in calculation rules was fixed to now start the calculation correctly.</td>
+<td style="text-align:left">COE-544</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The Andon board was fixed to include the default interval in the dropdown to switch between resolutions. This issue is related to COE-443.</td>
+<td style="text-align:left">COE-385</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The data chart on the machine dashboard was fixed to use correct UTC start date depending on the actual timezone. The wrong start date led to data gaps in the chart at the beginning of the selected timeframe. Also the data chart now includes the last measurement before the selected timeframe to ensure measurements are displayde for the entire selected timeframe without any gaps at the beginning of the selected timeframe.</td>
```

Comment:
"fixed to use correct UTC start date" -> "fixed to use the correct UTC start date"
"displayde" -> "displayed"

Reply:
@thomaswinkler @MarkJMReynolds-SAG Done